src/Symfony/Component/Workflow/TransitionBlocker.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow;
+
+/**
+ * A reason why a transition cannot be performed for a subject.
+ */
+final class TransitionBlocker
+{
+    const BLOCKED_BY_MARKING = '19beefc8-6b1e-4716-9d07-a39bd6d16e34';
```

Comment:
What are these?

Reply:
It was in the initial PR. 

It's used to identify a blocker by a "machine". It's the same approach as in the Validator component.

Reply:
I would make them `public` explicitly